lib/ansible/modules/cloud/amazon/aws_step_functions_state_machine_execution.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/python
+# Copyright (c) 2019, Prasad Katti (@prasadkatti)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: aws_step_functions_state_machine_execution
+
+short_description: Start or stop execution of an AWS Step Functions state machine
+
+version_added: "2.10"
+
+description:
+    - Start or stop execution of a state machine in AWS Step Functions.
+
+options:
+    action:
+        description: Desired action (start or stop) for a state machine execution
```

Comment:
niggle:

```suggestion
        description: Desired action (start or stop) for a state machine execution.
```

Descriptions should end with a "."
https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#documentation-block